WikiWatershed/model-my-watershed
# Add support for Ubuntu Xenial (16.04) on staging

## Overview

- Configure Gunicorn to log errors to syslog
- Configure rsyslog to forward Celery logs to syslog
- Configure rsyslog to forward syslog to Papertrail
- Remove references to monitoring service
- Reconcile VPC and DataPlane stack drift
- Reconcile `/var/lib/jenkins/.aws/staging.yml`

Resolves #3103 

## Testing Instructions

See that the CloudFormation stacks are in-sync:

![image](https://user-images.githubusercontent.com/1774125/60454093-eeb0ad80-9c00-11e9-85a9-3687763c3e7b.png)

Verify `feature/jrb/xenial-staging` is on staging:

 ```bash
maiden:~ rbreslow$ http --body https://staging.modelmywatershed.org/version.txt
origin/feature/jrb/xenial-staging 1.24.2-54-g4691c0a833f5e4ed07e080fff4f6ac675ac7e9d8
```

Select a HUC-8 unit and view its analysis:

![image](https://user-images.githubusercontent.com/1774125/60453938-8eba0700-9c00-11e9-8362-7af6d4c03890.png)

See the [Papertrail logs](https://papertrailapp.com/groups/14907082/events):

![image](https://user-images.githubusercontent.com/1774125/60453981-a72a2180-9c00-11e9-9260-c507cc3e603d.png)
